x-pack/platform/packages/shared/kbn-langchain/server/language_models/simple_chat_model.test.ts

```diff
@@ -252,6 +257,11 @@ describe('ActionsClientSimpleChatModel', () => {
 
       expect(rest).toEqual({
         temperature: 0,
+        model: undefined,
```

Comment:
I do not know why `model: undefined` suddenly had to be added, it's not part of my original changes..

Reply:
Did you add anything with your commit around it @pgayvallet

Reply:
AFAIK I did not